src/components/ImageUploader/ImageUploader.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from 'react';
+import ImageUploading, { ImageListType } from 'react-images-uploading';
+import './ImageUploader.css';
+
+interface ImageUploaderProps {
+  createDirectUpload: ({}) => void;
+}
+
+export const ImageUploader: React.FC<ImageUploaderProps> = ({
```

Comment:
Good work breaking this into it's own component, should make Modal much cleaner!